README.md

```diff
@@ -7,27 +7,179 @@
 
 # scrapbook
 
-A library for recording a notebook's data values and reading these recorded data
-values at a later time.
+A library for recording a notebook's data values and reading these recorded data values at a later time.
 
-**scrapbook** provides a robust and flexible recording schema. This library is
-planned to replace [papermill](https://papermill.readthedocs.io)'s existing
-`record` functionality.
+Namely scrapbook lets you:
+- **persist** data in a notebook
+- **highlight** named displays in notebooks
+- **recall** any persisted scrap of data or highlight
+- **summarize collections** of notebooks
 
 ## Use case
 
 Notebook users may wish to record data produced during a notebook execution.
 This recorded data can then be read to be used at a later time or be passed to
 another notebook as input.
 
-## papermill's record feature
+## API Calls
+
+Scrapbook adds a few basic api commands which enable saving and retrieving data.
+
+### glue
+
+Records a value (scrap) in the given notebook cell.
+
+The recorded value can be retrieved during later inspection of the output notebook.
 
-### Background
+```python
+sb.glue("hello", "world")
+sb.glue("number", 123)
+sb.glue("some_list", [1, 3, 5])
+sb.glue("some_dict", {"a": 1, "b": 2})
+sb.glue("non_json", df, 'arrow')
+```
+
+The scrapbook library can be used later to recover recorded values by reading the output notebook
+
+```python
+nb = sb.read_notebook('notebook.ipynb')
+nb.scraps
```

Comment:
I like `nb.scraps` and below the docs say `nb.highlights()`. I think this should be consistent (IMHO without parentheses)

Reply:
Yep. Was just a typo in the docs -- good catch